vendor/github.com/docker/go-connections/tlsconfig/config.go

```diff
@@ -83,6 +88,65 @@ func certPool(caFile string) (*x509.CertPool, error) {
 	return certPool, nil
 }
 
+func IsErrEncryptedKey(err error) bool {
+	return err != nil && strings.Contains(err.Error(), encryptedPrivateKeyMessage)
+}
```

Comment:
More concerning, this is a change to a vendored file but I don't see a corresponding update to `vendor.conf`.